client/src/Component/MainTableStyle.css

```diff
@@ -0,0 +1,12 @@
+.styled-table {
+    border-collapse: collapse;
+    margin: 25px 0;
+    font-size: 0.9em;
+    font-family: sans-serif;
+    min-width: 400px;
+    box-shadow: 0 0 20px rgba(0, 0, 0, 0.15);
```

Comment:
```suggestion
    margin: 1rem 0;
    font-size: 0.9em;
```
- It looks like `border-collapse: collapse` isn't doing anything—let's remove it
- We should use relative units for margin
- The font family is already sans serif so we can remove this
- We don't need `min-width` because the table will always be big
- We don't need box shadow here—tables usually don't have box shadow

Reply:
I got it Coung, changed and removed